user_docs/md2html.py

```diff
@@ -145,36 +133,29 @@ def _generateSanitizedHTML(md: str, isKeyCommands: bool = False) -> str:
 	return htmlOutput
 
 
-def md2html_actionFunc(
-	target: list[SCons.Node.FS.File],
-	source: list[SCons.Node.FS.File],
-	env: SCons.Environment.Environment,
-):
-	isKeyCommands = target[0].path.endswith("keyCommands.html")
-	isUserGuide = target[0].path.endswith("userGuide.html")
-	isDevGuide = target[0].path.endswith("developerGuide.html")
-	isChanges = target[0].path.endswith("changes.html")
-
-	with open(source[0].path, "r", encoding="utf-8") as mdFile:
+def main(source: str, dest: str, lang="en", docType=None):
+	print(f"Converting {docType or 'document'} at {source} to {dest}, {lang=}")
+	isUserGuide = docType == "userGuide"
+	isDevGuide = docType == "developerGuide"
+	isChanges = docType == "changes"
+	isKeyCommands = docType == "keyCommands"
+	if docType and not any([isUserGuide, isDevGuide, isChanges, isKeyCommands]):
+		raise ValueError(f"Unknown docType {docType}")
+	with open(source, "r", encoding="utf-8") as mdFile:
 		mdStr = mdFile.read()
 
-	mdStr = _replaceNVDATags(mdStr, env)
-
 	with io.StringIO() as mdBuffer:
 		mdBuffer.write(mdStr)
 		title = _getTitle(mdBuffer, isKeyCommands)
 
-	lang = pathlib.Path(source[0].path).parent.name
-	if isDevGuide and lang == "developerGuide":
-		# Parent folder in this case is the developerGuide folder in project docs
-		lang = "en"
+	lang = pathlib.Path(source).parent.name
 
 	if isUserGuide or isDevGuide:
-		extraStylesheet = '<link rel="stylesheet" href="numberedHeadings.css">'
+		extraStylesheet = ""
 	elif isChanges or isKeyCommands:
 		extraStylesheet = ""
```

Comment:
It seems like `extraStylesheet` is no longer used. Can it be removed?

Reply:
If this can be removed, I think we can remove doc type checking and simplify a lot of logic